crates/daphne-worker/src/durable/agg_job_response_store.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2025 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+//! Durable Object for storing the result of an aggregation job.
+
+use super::{req_parse, GcDurableObject};
+use crate::int_err;
+use daphne::protocol::ReadyAggregationJobResp;
+use daphne_service_utils::durable_requests::bindings::{
+    self, agg_job_response_store, DurableMethod as _,
+};
+use std::{sync::OnceLock, time::Duration};
+use worker::{js_sys, Env, Request, Response, Result, ScheduledTime, State};
+
+const AGGREGATE_RESPONSE_CHUNK_KEY_PREFIX: &str = "dap_agg_response_chunk";
+
+super::mk_durable_object! {
+    /// Where the aggregate share is stored. For the binding name see its
+    /// [`BINDING`](bindings::AggregateStore::BINDING)
+    struct AggJobResponseStore {
```

Comment:
FWIW we now write AggregationJobResp, AggregationJobResponse, and AggJobRespjonse. I think we'll thank ourselves later if we're at least consistent in this PR.